src/main/java/redis/clients/jedis/commands/SortedSetCommands.java

```diff
@@ -11,112 +11,608 @@
 
 public interface SortedSetCommands {
 
+  /**
+   * Add the specified member having the specified score to the sorted set stored at key. If member
+   * is already a member of the sorted set the score is updated, and the element reinserted in the
+   * right position to ensure sorting. If key does not exist a new sorted set with the specified
+   * member as sole member is created. If the key exists but does not hold a sorted set value an
+   * error is returned.
+   * <p>
+   * The score value can be the string representation of a double precision floating point number.
+   * <p>
+   * Time complexity O(log(N)) with N being the number of elements in the sorted set
+   * @param key
+   * @param score
+   * @param member
+   * @return Integer reply, specifically: 1 if the new element was added 0 if the element was
+   *         already a member of the sorted set and the score was updated
+   */
   long zadd(String key, double score, String member);
 
+  /**
+   * Similar to {@link SortedSetCommands#zadd(String, double, String) ZADD} with optional params.
+   * @param key
+   * @param score
+   * @param member
+   * @param params {@link ZAddParams}
+   */
   long zadd(String key, double score, String member, ZAddParams params);
 
+  /**
+   * Similar to {@link SortedSetCommands#zadd(String, double, String) ZADD} for multiple members.
+   * @param key
+   * @param scoreMembers
+   */
   long zadd(String key, Map<String, Double> scoreMembers);
 
+  /**
+   * Similar to {@link SortedSetCommands#zadd(String, Map<String, Double>) ZADD} with optional params.
```

Comment:
Method you're referring to does not contain any detail but refers to another one. Don't do that.